Assets/_BuildingBlocks/LayerStyles/Scripts/Metadata.cs

```diff
@@ -0,0 +1,10 @@
+using System.Runtime.Serialization;
+
+namespace Netherlands3D.LayerStyles
+{
+    [DataContract(Namespace = "https://netherlands3d.eu/schemas/projects/layers/styling", Name = "Metadata")]
```

Comment:
Aangezien dit in de styling thuis hoort, is het handig om hier StylingMetadata van te maken om ambiguiteit te voorkomen?

Reply:
Dat zit al in de namespace van de class en de typing van het DataContract